man/tpm2_import.1.md

```diff
@@ -56,6 +56,11 @@ These options control the key importation process:
   * **-A**, **--object-attributes**=_ATTRIBUTES_:
     The object attributes, optional.
 
+  * **--passin**=_OSSL\_PEM\_FILE\_PASSWORD_
+    An optional password for an Open SSL (OSSL) provided input file. It mirrors the -passin option of
+    OSSL and is known to support the pass, file, env, fd and plain password formats of openssl.
+    (see *man(1) openssl*) for more.
+
```

Comment:
We've documented the passin formats in the code but not in the manual pages, it might make sense to reproduce the code comments describing passin here?

Reply:
Sure, I guess I could do that. This was more of a note for me as I was implementing. This is a copy + paste from OSSL and I don't want to be in violation of any license terms.